glide.lock

```diff
@@ -25,21 +25,17 @@ imports:
   - util/runes
   - util/strings
 - name: github.com/gocql/gocql
-  version: 33a5f3c1bcc2c421b3221c5858312afb141bf605
+  version: dd47639f787e8583e1a13e17e20a9f6c4332bc29
```

Comment:
check the version for gocql

Reply:
Have we finalized the version that breaks our client? 

I can pin the version in yaml for now, until we are safe to upgrade. Is there a ticket to track it?

Reply:
I reverted the gocql version and pined it in yaml file